zygosome.go

```diff
@@ -549,6 +582,39 @@ func NewZygoNucleus(h *Holochain, code string) (n Nucleus, err error) {
 			return &result, nil
 		})
 
+	z.env.AddFunction("send",
+		func(env *zygo.Glisp, name string, zyargs []zygo.Sexp) (zygo.Sexp, error) {
+			a := &ActionSend{}
+			args := a.Args()
+			err := zyProcessArgs(args, zyargs)
+			if err != nil {
+				return zygo.SexpNull, err
+			}
+
+			a.to, err = peer.IDB58Decode(args[0].value.(Hash).String())
+			if err != nil {
+				return zygo.SexpNull, err
+			}
+
+			msg := args[1].value.(map[string]interface{})
+			var j []byte
+			j, err = json.Marshal(msg)
+			if err != nil {
+				return zygo.SexpNull, err
+			}
+
+			a.msg.ZomeType = z.zome.Name
+			a.msg.Body = string(j)
+
+			var r interface{}
+			r, err = a.Do(h)
+			var resp zygo.Sexp
+			if err == nil {
+				resp = &zygo.SexpStr{S: r.(string)}
+			}
+			return makeResult(env, resp, err)
+		})
+
```

Comment:
Hm.. There is quite some duplicated code between the ribosomes. The send function in the JS ribosome looks almost the same. And actually that is the case for most ribosome functions. Would like to see some way to reduce the redundancy here in general, but that is maybe out of the scope of this PR.